pwndbg/dbg/gdb.py

```diff
@@ -484,6 +496,18 @@ def selected_inferior(self) -> pwndbg.dbg_mod.Process | None:
     def is_gdblib_available(self):
         return True
 
+    @override
+    def has_event_type(self, ty: pwndbg.dbg_mod.EventType) -> bool:
+        # Currently GDB supports all event types.
+        return True
+
+    @override
+    def event_handler(
+        self, ty: pwndbg.dbg_mod.EventType
+    ) -> Callable[[Callable[..., T]], Callable[..., T]]:
+        # Just call into the gdblib handler.
+        return GDBLIB_EVENT_MAPPING[ty]
```

Comment:
It would be better to do if elif else imho instead of introducing a new dict object unless it has another purpose

Reply:
Yeah, now that I think of it, I don't think the supported event list is ever gonna change. When I fist picked the dictionary I'd intended the dictionary for a possible case where the types of supported events could change over startup. But now with it implemented I don't think it makes sense to keep it. Let me change it.